### PR TITLE
activate Travis build + add maven coordinate 

### PR DESCRIPTION
**Pull request by ghost**
#4 activate travis build 
#2 travis is needed for publishing in maven central

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,64 +1,73 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-	<groupId>ContentSecurityPolicyFilter</groupId>
-	<artifactId>ContentSecurityPolicyFilter</artifactId>
-	<version>2.0.0</version>
+    <groupId>be.nbb.csp</groupId>
```
Comment (dsebastien) [inline pom.xml]: Let's use be.nbb.security.csp instead

File: pom.xml [hunk 2]
```diff
@@ -1,64 +1,73 @@
...
+    <artifactId>ContentSecurityPolicyFilter</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
 
-	<name>ContentSecurityPolicyFilter</name>
+    <name>ContentSecurityPolicyFilter</name>
```
Comment (dsebastien) [inline pom.xml:9]: To avoid tabs vs spaces nightmare we'll add editorconfig: #8

**Review by dsebastien — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (ghost) [reply pom.xml:9]: will make another PR to fix that

File: pom.xml (hunk 1 above)
Comment (ghost) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (ghost) [reply pom.xml:9]: resolved with #8 in PR #10

**Comment by dsebastien** [comment]
@maximeSauvageRD could you create a new PR for the Travis config?
Then we'll just close and forget this PR..

**Comment by ghost** [comment]
what about the maven coordonate ?

**Comment by dsebastien** [comment]
That too :)

**Comment by ghost** [comment]
in the same PR of travis or in another PR ?

**Comment by dsebastien** [comment]
Can be in the same I don't mind

**Comment by ghost** [comment]
ok juste another PR with basically the same content ? :p

**Comment by dsebastien** [comment]
Look at the commit list of this PR. It’s 7 commits so it doesn’t make sense for the project history. Each pr should only contain relevant commits

**Comment by dsebastien** [comment]
Anyways let’s move forward we can’t spend a week on this :)

**Pull request merged**